### DISCUSSION TITLE
Chart 4.0 Beta will coming soon!

### DISCUSSION THREAD

**Discussion by jung-han** [nhn/tui.chart#427] [General]
# 🙋‍♂️ Chart 4.0 Beta is coming! 🙋‍♀️

![image](https://user-images.githubusercontent.com/35371660/97143156-75e64500-17a5-11eb-9e73-d1201d07c4d7.png)

Finally, the beta version of the 4.0 major version update is scheduled for late November!
Let's wait and see what's going on in 4.0.

```
TLDR:
* "svg" to "canvas"
* Remove all dependencies
* Live data update 
* Responsive layout
* More options, More themes, More APIs
* and much more
```

**`svg` to `canvas`**

Chart, which was developed based on svg in version 3, was reborn as canvas from version 4. It became a lighter chart by removing both the `raphael` and the `tui-code-snippet` dependencies.

**features**

Responsive layout and live data features are supported. And, a lot of theme options are added to help you develop charts that fit your purpose. In addition, parameters and API structures that were not intuitive have been modified, and more APIs have been added.

In addition, the repository will be supported in monorepo so that issues will be managed in one place, and the map chart provided with version 3 will be removed from the chart and will be newly appeared as an advanced`TOAST UI Map Chart`.

There are many other changes, so please wait a little longer and look forward to it!

**Comment by shiren** [comment]
👍

**Comment by lja1018** [comment]
👍 👍

**Comment by chchoing88** [comment]
👍👏

**Comment by jinwoo-kim-nhn** [comment]
👍 👍 👍

**Comment by dongsik-yoo** [comment]
👍 👍 👍👍

**Comment by jwlee1108** [comment]
👍 👍👍

**Comment by jung-han** [comment]
Sorry for the late news.
Due to some delay, there will be beta news in mid-December!
I'm sorry again and hope you wait a little longer!
